src/main/converter/opds.ts

```diff
@@ -176,6 +176,14 @@ export class OpdsFeedViewConverter {
             navigation = feed.Navigation.map((item) => {
                 return this.convertOpdsLinkToView(item);
             });
+
+            // concatenate all relative path to an absolute URL path
+            navigation = navigation.map((nav) => {
+                if (nav.url && !/[a-zA-Z]+\:\/\//.exec(nav.url)) {
```

Comment:
The lack of boundary "beginning" indicator `^` in the `RegExp` (i.e. before the pattern matcher for the URL scheme / protocol) means that there is a possibility of false positives, for instance:

`node -e "const url = './catalog/search://test'; console.log(/[a-zA-Z]+:\/\//.exec(url));"`
=>
```
[ 'search://',
  index: 10,
  input: './catalog/search://test',
  groups: undefined ]
```

versus (just added `^`):

`node -e "const url = './catalog/search://test'; console.log(/^[a-zA-Z]+:\/\//.exec(url));"`
=>
`null` (correct expected result)

(note: `^` means that the input `url` string *must* be trimmed from whitespace characters on the leading edge!)

Reply:
thanks Daniel
Done https://github.com/readium/readium-desktop/pull/639/commits/bd5fb1ef779b422b850121d3a44ad2600d7d347f